modules/admin/proc-migrating-databases-to-an-external-server.adoc

```diff
@@ -0,0 +1,93 @@
+
+[NOTE]
+====
+For quick migration, you can also use this link:https://github.com/janus-idp/operator/blob/main/hack/db_copy.sh[database copy script].
+====
+
+.Prerequisites
+
+* You have installed the link:https://www.postgresql.org/docs/current/app-pgdump.html[pg_dump] and link:https://www.postgresql.org/docs/current/app-psql.html[psql] utilities on your local machine.
+* For data export, you have the PGSQL user sufficient privileges to make a full dump of local databases.
+* For data import, you have the PGSQL user sufficient admin privileges to create an external database and populate it with database dumps.
+
+.Procedure
+
+. Configure port forwarding for the local PostgreSQL database pod by running the following command on a dedicated terminal: 
++
+[source,terminal]
+----
+oc port-forward -n <your-namespace> <pgsql-pod-name> <forward-to-port>:<forward-from-port>
+----
+Where:
+* The `<pgsql-pod-name>` variable denotes the name of a PostgreSQL pod with the format `backstage-psql--<_index>`.
+* The `<forward-to-port>` variable denotes the port of your choice to forward PostgreSQL data to.
+* The `<forward-from-port>` variable denotes the local PostgreSQL instance port, usually `5432`.
++
+.Example: Configuring port forwarding
+[source,terminal]
+----
+oc port-forward -n backstage backstage-psql-backstage1-0 15432:5432
+----
+
+. Make a copy of the `db_copy.sh` script and edit the following details based on your configuration:
+
+* `to_host`: The destination host name, for example, `<db-instance-name>.rds.amazonaws.com`
+* `to_port`: The destination port that is usually `5432`
+* `to_user`: The destination server username, for example, `postgres`
+* `from_host`: The source host name that is usually `127.0.0.1`
+* `from_port`: The source port number, that is, the `<forward-to-port>` variable 
+* `from_user`: The source server username, for example, `postgres`
+* `allDB`: The name of databases to import in double quotes separated by spaces, for example, `("backstage_plugin_app" "backstage_plugin_auth" "backstage_plugin_catalog" "backstage_plugin_permission" "backstage_plugin_scaffolder" "backstage_plugin_search")`
+
+. Create a destination database for copying the data:
++
+[source,terminal]
+----
+/bin/bash TO_PSW=<destination-db-password> /path/to/db_copy.sh
+----
+Where,
+* The `<destination-db-password>` variable denotes the password to connect to the destination database.
++
+[NOTE]
+====
+You can stop port forwarding when copying of the data is complete. Also, if you are dealing with large databases, go to the link:https://www.postgresql.org/docs/current/backup-dump.html#BACKUP-DUMP-LARGE[site] and see the details to use the compression tools. 
+====
+
+. Reconfigure your `Backstage` custom resource (CR). For more information, see link:{LinkAdminGuide}#proc-configuring-postgresql-instance-using-operator_admin-rhdh[Configuring an external PostgreSQL instance using the Operator]. 
```

Comment:
This should be Developer Hub CR, we also have procedure describing how to configure RHDH CR: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.1/html-single/administration_guide_for_red_hat_developer_hub/index#proc-config-rhdh-custom-resource_admin-rhdh

Reply:
I think we should use the exact CR name. the link mentioned here also point to the `Backstage` CR, not the `Developer Hub` CR. throughout the doc, we have use the name `Backstage` CR, not `Developer Hub` CR.